src/main/java/org/fundacion/jala/converter/view/MainInterface.java

```diff
@@ -33,21 +33,28 @@
     private MetaDataInterface metaData;
     private AudioConverterInterface audio;
     private TextExtractorInterface textExtractor;
+    private final int alignLabelStyle = 0;
```

Comment:
are these variables or constants? if they are constants they should be in capital letters.

Reply:
In a code convention, we decided that constants would be written in lowercase :)